gnupg-pkcs11-scd/command.c

```diff
@@ -30,6 +30,7 @@
 
 #include "common.h"
 #include "strgetopt.h"
+#include <pkcs11-helper-1.0/pkcs11.h>
```

Comment:
why is this required?

Reply:
I added this for things like `CKR_OK` because it wasn't clear it would otherwise be included.